### DISCUSSION TITLE
No chat logs in the extension

### DISCUSSION THREAD

**Discussion by tsilvs** [benf2004/Bing-Chat-History#46] [General]
Extension was running the whole time.

![image](https://github.com/benf2004/Bing-Chat-History/assets/26058495/525553f7-f5ed-4fd8-9c31-94b33caf8ada)
![image](https://github.com/benf2004/Bing-Chat-History/assets/26058495/dcd5752c-1f28-4d37-a42c-7de4d68599af)

**Comment by LJ-netizen** [comment]
The Bing Chat History extension used to work excellently in the Microsoft Edge browser; however as of sometime between late July 2024 and now, August 8, 2024, it has ceased to work. It no longer copies or records any conversation between Copilot and me. All that the Bing Chat History extension now displays are my older conversations (which still can be viewed); however, when I reinstall the extension all that is continually seen is the screenshot above (as shown by "tsilvs").

**Comment by LJ-netizen** [comment]
**Update:** The Bing Chat History extension in Microsoft Edge browser is once again working. I went into Windows Settings | Apps | Apps and Features, and scrolled down to Microsoft Edge, clicked on it then clicked on its Modify button, and next clicked on 'Repair' in the small window which will open. After it completed the reinstall (no data was lost), everything once more worked as it ought. Two thumbs up.
